sdk/schemaregistry/azure-schemaregistry-avroserializer/README.md

```diff
@@ -27,7 +27,22 @@ To use this package, you must have:
 * Python 2.7, 3.6 or later - [Install Python][python]
 
 ### Authenticate the client
-Interaction with Schema Registry Avro Serializer starts with an instance of AvroSerializer class. You need the fully qualified namespace, AAD credential and schema group name to instantiate the client object.
+Interaction with Schema Registry Avro Serializer starts with an instance of AvroSerializer class. The client constructor takes the fully qualified namespace, schema group name, and and Azure Active Directory credential:
```

Comment:
the serializer takes SRClient and group name.. so actually it's not the SR Serializer being authenticated but the SRClient passsed in which makes me wonder if we need this section or not

Reply:
yeah that's a good point